profile/attribute/image_slice_attribute.go

```diff
@@ -11,7 +12,7 @@ import (
 // ImageSliceAttribute is a Yoti attribute which returns a slice of images as its value
 type ImageSliceAttribute struct {
 	attributeDetails
-	value []*media.Image
+	value []media.Media
```

Comment:
Do you think it's better to not use pointers with these images/media then?

Reply:
Media is an interface, you would not normally have pointers to interface

Reply:
Ah yes, makes sense